cogs/greetings.py

```diff
@@ -20,3 +20,14 @@ async def man(self, ctx: Context):
     async def l(self, ctx: Context):
         # Sends a message to the channel using the Context object.
         await ctx.channel.send("L")
+
+    @commands.command()
+    async def choose(self, ctx: Context, *args):
```

Comment:
Give typing to args

`*args: str`

That way we know that strings are being inputted